### DISCUSSION TITLE
How to remove the icon from the header?

### DISCUSSION THREAD

**Discussion by kostrykin** [Textualize/textual#4909] [Q&A] [answered]
I can't figure out how to remove the icon from the header.

I tried `Header(icon = '')` which makes it invisible, but it is still highlighted when hovered with the mouse cursor.

**Comment by kbernhagen** [comment]
I believe you need to disable the command palette.

https://textual.textualize.io/guide/command_palette/#disabling-the-command-palette

**Reply by TomJGooding** [reply]
Disabling the command palette will still show the icon and highlighted when hovered though.

**Comment by TomJGooding** [comment, marked as answer]
The `Header` doesn't provide an option to remove the icon, but you could do something like this:

```python
from textual.app import App, ComposeResult
from textual.widgets import Header


class ExampleApp(App):
    def compose(self) -> ComposeResult:
        yield Header()

    def on_mount(self) -> None:
        self.query_one("HeaderIcon").visible = False


if __name__ == "__main__":
    app = ExampleApp()
    app.run()
```

Or hide it using CSS like this:

```css
    HeaderIcon {
        visibility: hidden;
    }
```

**Comment by willmcgugan** [comment]
Untested, but this CSS should hide the icon:

```css
HeaderIcon {
    display: none;
}
```
